### DISCUSSION TITLE
'database is closed' 500 response when using public API

### DISCUSSION THREAD

**Discussion by theetrain** [Budibase/budibase#6413] [Help]
When trying to connect to my application via the public API, I receive a 500 error, which occurs after this request:

Request:
```sh
# Note, I use --insecure because my host doesn't have a certificate
curl --insecure --request GET \
  --url "$BB_HOST/api/public/v1/applications/$BB_APP_ID" \
  --header "Accept: application/json" \
  --header "x-budibase-api-key: $BB_API_KEY"
```

Response:
```sh
{"message":"database is closed","status":500}
```

I'm able to interact with my Budibase application and database on my hosted instance (v1.0.196), so I'm unsure why it reads 'database is closed'. The public API has been working for the past few weeks for me (likely on older versions of Budibase, but possibly v1.0.196 as well), but I checked on it this week and for some reason I face this issue.

**Comment by melohagan** [comment]
Hi @theetrain 

I would double check that the `BB_APP_ID` is definitely correct and matching the app_id of your hosted instance.

I would also test out both the dev and published URLs
